### PR TITLE
Add email subscription status and topics

### PR DESCRIPTION
**Pull request by katiecrane**
### What's Changed

Added `email_subscription_status` and `email_subscription_topics` to user profile. I separated subscriptions out into a separate sections because I thought the list was getting a little long and hard to parse, but as you know I am _not_ a designer so happy to leave everything in one list if folks like that better!

User with no email topics:
image

User with email topics:
image

### CODE REVIEW

File: resources/views/users/show.blade.php [hunk 1]
```diff
@@ -11,8 +11,15 @@
...
             <div class="container__block -half profile-settings">
-                <h3>Profile</h3>
-                @include('users.partials.profile')
+                <div class="container -padded">
+                    <h3>Profile</h3>
+                    @include('users.partials.profile')
+                </div>
+                <div class="container -padded">
+                    <h3>Subscriptions</h3>
+                    @include('users.partials.subscriptions')
```
Comment (DFurnes) [inline resources/views/users/show.blade.php]: Is this partial missing from the PR? 🕵️

Comment (katiecrane) [reply resources/views/users/show.blade.php]: agh well I was trying to push it up but GitHub has started asking me for a password each time and sometimes I miss it! it's here now!

Comment (DFurnes) [reply resources/views/users/show.blade.php]: Ah, frustrating! You should be able to fix that [by configuring a helper](https://help.github.com/en/articles/caching-your-github-password-in-git) or updating your git remote to use SSH for authentication instead (which is what I do).
